src/Symfony/Component/Config/Resource/ClassExistenceResource.php

```diff
@@ -191,15 +191,17 @@ public static function throwOnRequiredClass($class, \Exception $previous = null)
             }
 
             $props = [
-                'file' => $trace[$i]['file'],
-                'line' => $trace[$i]['line'],
+                'file' => isset($trace[$i]['file']) ? $trace[$i]['file'] : null,
```

Comment:
I'm just wondering why not using `$trace[$i]['file'] ?? null` instead? is it for backword comptability with PHP 5.6?

Reply:
I see it's 3.4 with support for php 5.4 👍